lib/bugsnag/notification.rb

```diff
@@ -181,7 +181,7 @@ def deliver
         Bugsnag.warn "No API key configured, couldn't notify"
         return
       elsif api_key !~ API_KEY_REGEX
-        Bugsnag.warn "Your API key (#{api_key}) is not valid, couldn't notify"
```

Comment:
Do you think that it is still a dangerous log line, say if someone accidentally set the Bugsnag API key to another API key or malformed API key?
Would the message not give enough information without logging the API key?

If not, I'll not remove this log line but would prefer too.